packages/server/src/api/ampEpicRouter.ts

```diff
@@ -143,30 +149,33 @@ export const buildAmpEpicRouter = (
                         hideReminderCta: false,
                         hideReminderForm: false,
                     },
-                    choiceCards: epic.showChoiceCards
-                        ? {
-                              choiceCardSelection: {
-                                  frequency: defaultChoiceCardFrequency,
-                                  amount:
-                                      choiceCardAmountsData[defaultChoiceCardFrequency].amounts[1],
-                              },
-                              amounts: {
-                                  ONE_OFF: choiceCardAmountsData['ONE_OFF'].amounts.slice(0, 2),
-                                  MONTHLY: choiceCardAmountsData['MONTHLY'].amounts.slice(0, 2),
-                                  ANNUAL: choiceCardAmountsData['ANNUAL'].amounts.slice(0, 2),
-                              },
-                              choiceCardLabelSuffix: {
-                                  ONE_OFF: '',
-                                  MONTHLY: ' per month',
-                                  ANNUAL: ' per year',
-                              },
-                              classNames: {
-                                  choiceCard: 'epicChoiceCard',
-                                  choiceCardSelected: 'epicChoiceCard epicChoiceCardSelected',
-                              },
-                              currencySymbol: getLocalCurrencySymbol(countryCode),
-                          }
-                        : false,
+                    choiceCards:
+                        epic.showChoiceCards &&
+                        choiceCardAmountsData != null &&
+                        defaultChoiceCardFrequency != null
+                            ? {
+                                  choiceCardSelection: {
+                                      frequency: defaultChoiceCardFrequency,
+                                      amount: choiceCardAmountsData[defaultChoiceCardFrequency]
+                                          .amounts[1] as number,
+                                  },
+                                  amounts: {
+                                      ONE_OFF: choiceCardAmountsData['ONE_OFF'].amounts.slice(0, 2),
```

Comment:
What's the significance of the 2 here? Would it be worth pulling out to a const to give it a name?

Reply:
Was existing code which GitHub has decided to display as new code. Epic choice cards only have space to display 2 amounts (+ the `other amounts` button) in any contributionType tab, whereas the landing page choice cards can display an infinite number of suggested price points. Current functionality is to display the lowest two price points (SAC code sorts the prices lowest -> highest in the array, when Marketing creates/updates the amounts test). Does that explain why?

Reply:
It does, thanks! Perhaps we could encode this in a const:

```js
const AMOUNTS_CHOICE_LIMIT = 2;
```

and then later:

```js
  ONE_OFF: choiceCardAmountsData['ONE_OFF'].amounts.slice(0, AMOUNTS_CHOICE_LIMIT)
```

Reply:
(But if the code wasn't added in this PR then fine to leave for another time).